packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -54,7 +54,10 @@ export function SquadHeaderBar({
         )}
       >
         <Button
-          className="btn-primary"
+          className={classNames(
+            'btn-primary',
+            copyLinkTutorial.isActive && 'highlight-pulse',
```

Comment:
This is the wrong prompt.
These pulse animations are seperate from the other CTA.

Basically the pulse should show DURING the tour, and the other ones AFTER the tour.
Let me know if you want me to show you quick?

Reply:
Oh, cool, when they are at that stage. Let me update it.